PWGDQ/Tasks/tableReader.cxx

```diff
@@ -966,6 +966,15 @@ void DefineHistograms(HistogramManager* histMan, TString histClasses)
     if (classStr.Contains("Track")) {
       if (classStr.Contains("Barrel")) {
         dqhistograms::DefineHistograms(histMan, objArray->At(iclass)->GetName(), "track", "its,tpcpid,dca,tofpid");
+        if (classStr.Contains("PIDCalibElectron")) {
+          dqhistograms::DefineHistograms(histMan, objArray->At(iclass)->GetName(), "track", "electron");
```

Comment:
this would not work, because what you have now in the HistogramsLibrary is now requiring the subgroup name to contain both "postcalib" and "electron". So you should use something like "postcalib_electron".

Reply:
Hi,Ionut 
Yes, right we have, in the cut selection "cfgTrackCuts": "PIDCalibElectron,PIDCalibPion,PIDCalibProton",
Cheers
Xiaozhi